_posts/2020-08-19-#19509.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: pr
+date: 2020-08-19
+title: "Per-peer Message Logging"
+pr: 19509
+authors: [troygiorshev]
+components: ["p2p", "utils/log/libs"]
+host: troygiorshev
+status: upcoming
+commit: 4fc2e3906
+---
+
+## Notes
+
+* [PR 19509](https://github.com/bitcoin/bitcoin/pull/19509) is a feature PR, adding per-peer message logging to Bitcoin Core.
+
+* Nodes on the Bitcoin network communicate using a variety of [messages](https://btcinformation.org/en/developer-reference#p2p-network).  Now these messages can be logged for easy review, in order and on a per-peer basis.
```

Comment:
maybe s/Now/This PR proposes to permit/ and s/order/chronological order/ (if that's the case)

Reply:
reworded